OSDOCS-3039: Improve kubelet CA update for Paused Machine Config Pools (#45313)
openshift/openshift-docs: modules/compliance-applying.adoc

Note that when the Machine Config Operator applies a new `MachineConfig` object to nodes in a pool, all the nodes belonging to the pool are rebooted. This might be inconvenient when applying multiple remediations, each of which re-renders the composite `75-$scan-name-$suite-name` `MachineConfig` object. To prevent applying the remediation immediately, you can pause the machine config pool by setting the `.spec.paused` attribute of a `MachineConfigPool` object to `true`.

[NOTE]
====

> Make sure the pools are unpaused when the CA cert rotation happens. If the MCPs are paused, the cert rotation does not happen, which causes the cluster to become degraded and causes failure in multiple `oc` commands, including but not limited to `oc debug`, `oc logs`, `oc exec`, and `oc attach`. You receive alerts in the Alerting UI of the {product-title} web console if an MCP is paused when the certificates are rotated.

Reviewer: What @rioliu-rh  is saying is technically correct. As far as the nodes are concerned, the "upstream" certificate rotation does happen, and the certificates but it doesn't "happen" for the nodes. Whether we want to be that specific/granular in terms of what "happens" where,I don't know?

> Make sure the pools are unpaused when the CA cert rotation happens. If the MCPs are paused, the cert rotation does not happen, which causes the cluster to become degraded and causes failure in multiple `oc` commands, including but not limited to `oc debug`, `oc logs`, `oc exec`, and `oc attach`. You receive alerts in the Alerting UI of the {product-title} web console if an MCP is paused when the certificates are rotated.

Author: @rioliu-rh @jkyros Is this more accurate?
_Make sure the pools are unpaused when the CA cert rotation happens. If the MCPs are paused, **the MCO cannot push the newly-rotated certs to those nodes**,_

> Make sure the pools are unpaused when the CA cert rotation happens. If the MCPs are paused, the cert rotation does not happen, which causes the cluster to become degraded and causes failure in multiple `oc` commands, including but not limited to `oc debug`, `oc logs`, `oc exec`, and `oc attach`. You receive alerts in the Alerting UI of the {product-title} web console if an MCP is paused when the certificates are rotated.

Reviewer 2: yes, looks good to me

> Make sure the pools are unpaused when the CA cert rotation happens. If the MCPs are paused, the MCO cannot push the newly-rotated certs to those nodes. This causes the cluster to become degraded and causes failure in multiple `oc` commands, including but not limited to `oc debug`, `oc logs`, `oc exec`, and `oc attach`. You receive alerts in the Alerting UI of the {product-title} web console if an MCP is paused when the certificates are rotated.

Reviewer 3: Technically, "including but not limited to" should have commas to set it off the parenthetical part ("including, but not limited to,"), but that would yield: "multiple `oc` commands, including, but not limited to, `oc debug`, `oc logs`, `oc exec`, and `oc attach`" which imo is pretty clunky. 

Might be simplest/most minimalist to just let "including: do the work as below:

```suggestion
Make sure the pools are unpaused when the CA certificate rotation happens. If the MCPs are paused, the MCO cannot push the newly rotated certificates to those nodes. This causes the cluster to become degraded and causes failure in multiple `oc` commands, including `oc debug`, `oc logs`, `oc exec`, and `oc attach`. You receive alerts in the Alerting UI of the {product-title} web console if an MCP is paused when the certificates are rotated.
```

Either way NB s/`newly-rotated`/`newly rotated` which is easy to miss in the little diff preview :slightly_smiling_face:
